custom-seldon-server/docker/base/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# From python docker image
+FROM python:3.7-slim
+WORKDIR /app
+
+# Install seldon-core
+RUN pip install seldon-core==1.13.1 werkzeug==2.0.3
```

Comment:
As far as I know werkzeug is WSGI lib, how we use with this showcase?

Reply:
When I use the `seldon-core` python package, I get the error that needs to install werkzeug. I think seldon-core needs to use the WSGI lib to do something.

Reply:
got it